src/Indicator.vala

```diff
@@ -541,6 +545,18 @@ public class Sound.Indicator : Wingpanel.Indicator {
 
         return true;
     }
+
+    private void update_tooltip () {
+        debug ("VolumeControl ready? %s".printf (volume_control.ready.to_string ()));
```

Comment:
This may spam debug output - probably not useful in final code.

Reply:
Yeah my bad. Was having an issue where `volume_control` was null hence the debug. I'll remove it. Thanks!